src/components/mermaidMap/SingleSiteMap/SingleSiteMap.js

```diff
@@ -169,19 +174,34 @@ const SingleSiteMap = ({
 
   const updateBenthicLayers = (dataLayerFromLocalStorage) =>
     setGeomorphicOrBenthicLayerProperty(map.current, 'atlas-benthic', dataLayerFromLocalStorage)
-  const placeMarkerButtonText =
-    formLatitudeValue && formLongitudeValue
-      ? language.pages.siteForm.replaceMarker
-      : language.pages.siteForm.placeMarker
 
   const handlePlaceMarkerClick = () => {
     setIsMarkerBeingPlaced(!isMarkerBeingPlaced)
+    hasLatLngChanged && setHasLatLngChanged(false)
```

Comment:
did you mean to wrap this in an if statement. instead of using `&&` (which will work coincidentally). 

You could also just call `setHasLatLngChanged(false)` without checking since internally React wont do anything if its already the value you are setting it to (I believe)

Reply:
co-pilot recommended this one ... tried it out and it worked - but I think it's clearer if I wrap in an if statement